AGP-24_Deprecate_0_5_Rinkeby_DAOs.md

```diff
@@ -0,0 +1,61 @@
+## Preamble
+
+    AGP: 24
+    Title: Deprecation of 0.5 Rinkeby DAOs
+    Author: Brett Sun
+    Type: Product / Communication
+    Status: Final, v1.0
+    Created: 2018-09-27
+    Last updated: 2018-09-27
+    Requires:
+
+## Abstract
+
+Updates to aragonOS (`aragonOS@4`) are backwards-incompatible (e.g. non-upgradeable) with the current base contracts and proxy contracts deployed on Rinkeby, affecting any DAOs created via the 0.5 [democracy kit](https://rinkeby.etherscan.io/address/0x344bc497128f1e373639fa0621a12331f65bdc7a) or [multisig kit](https://rinkeby.etherscan.io/address/0x876556bbea9e66e8580adfe1f960dc87c3652bfd).
+
+A decision has been made to deprecate these old DAOs, and communicate to users the extent of this deprecation as well as how to recreate their DAOs on Rinkeby.
+
+## Specification
+
+### Goals
+
+- Effectively communicate to the user base that this deprecation is happening, and allow them to respond
+- Make the switch over to the new contracts as painless as possible
+
+## Description
+
+`aragonOS@4` introduces a number of breaking changes from `aragonOS@3`. We'll be deploying these new, backwards-incompatible contracts on both Rinkeby and Mainnet.
+
+At the moment, most features should just work, but there are a few breaking changes:
+
+- Old users won't be able to upgrade to new versions of any contracts (either internal to aragonOS, like the Kernel or ACL, or any Apps), without there being internal state inconsistencies
+- Potentially ACL write functionality may not work ([aragon/aragonOS#223](https://github.com/aragon/aragonOS/issues/223))
+- Old users won't have access to the recovery vault functionality
+- Some functionality not currently exposed in the UI
+
+As part of the migration, a new root ENS registry, as well as Aragon Package Manager, will be deployed. A new root ENS registry will allow users to recreate DAOs with their old ENS names (although there is the issue with squatting these now known names).
+
+### Roadmap
+
+**Oct. 8/9:**
+
+- Deploy contracts based on `aragonOS@4` to Rinkeby, with new ENS and Aragon Package Manager instances
+- Provide a way to access the old frontend (e.g. old-app.aragonpm.eth)
```

Comment:
Something that came up is how long we should keep the old frontend up for. As a suggestion I'd say the end of March, 2018, so that we could say it had a full year of life as an :eagle:.